contrib/download-frozen-image-v2.sh

```diff
@@ -332,7 +343,7 @@ while [ $# -gt 0 ]; do
 						fi
 					done
 					if [ -z "$found" ]; then
-						echo >&2 "error: manifest for $maniArch is not found"
+						echo >&2 "error: manifest for $targetArchVariant is not found"
```

Comment:
[Shell parameter expansion](https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html) could be used instead of the `targetArchVariant` logic.
```suggestion
						echo >&2 "error: manifest for ${targetArch}${targetVariant:+/${targetVariant}} is not found"
```